### ISSUE TITLE
"syslog" file not located

### ISSUE THREAD

**Issue by RafaelLinux** [Sysinternals/SysmonForLinux#46]
Following this site instructions, I installed sysmonForLinux on openSUSE Tumbleweed.
I ran `sysmon` and it's working. However, log file "`/var/log/syslog`" doesn't appear and  even `sysmonLogView` doesn't show nothing.
Where is located that file in my system?

Thank you

**Comment by kesheldr** [comment]
Hello

It will be wherever the syslog messages go - maybe /var/log/messages or similar?

**Comment by RafaelLinux** [comment]
Yes, there exist that log there. And now, how to make sysmon to show stored data thru a window, like Plasma "System Monitor" or "KSysGuard"?

**Comment by RafaelLinux** [comment]
My fault then, I was wrong about how it works.

Thank you for clarify and spend your time with my question!!!
